tox.ini

```diff
@@ -13,9 +13,7 @@ setenv=
 skip_install = true
 envdir = {toxworkdir}/py39
 commands=
-    ; Force packaging even if setup.{py,cfg} haven't changed
-    rm -rf ./*.egg-info/
-    python setup.py coverage
+    python setup.py coverage {posargs}
```

Comment:
Since there was a change on `setup.py` to explicitly refer to `python3`, I wonder if we shouldn't also use `python3` here instead of `python`. wdyt?

Reply:
For sure. That should've been python3 to be uniform, fixed it here [9594cc9](https://github.com/kytos-ng/pathfinder/pull/22/commits/9594cc97777080243bda5999ed58a625d7d6fdd3), thanks for reviewing @italovalcy